section9_dictionary/project/README.md

```diff
@@ -0,0 +1,20 @@
+# Blind Auction      
```

Comment:
Please remove the entire section9 directory from the branch section8 since it will cause the merge conflict in future